datahub/main.py

```diff
@@ -1,16 +1,26 @@
 """Script for running Datahub API."""
+import logging
+import logging.config
 from typing import Any, Hashable
 
 from fastapi import FastAPI, HTTPException
 from pydantic import BaseModel
 
+from datahub.core.log_config import logging_dict_config
+
 from . import data as dt
 from .dsr import DSRModel, validate_dsr_arrays
 from .opal import OpalModel
 from .wesim import get_wesim
 
+logging.config.dictConfig(logging_dict_config)
+
 app = FastAPI()
 
+log = logging.getLogger("api_logger")
+log.debug("Logging is configured.")
+log.info("Datahub API is running...")
```

Comment:
I would move this into `__init__.py` so other modules can import it with `from . import log`